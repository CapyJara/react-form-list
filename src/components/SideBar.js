import React from 'react';
import PropTypes from 'prop-types';

function SideBar({ children }) {
  return (
    <nav>
      {children}
    </nav>
  );
}

SideBar.propTypes = {
  children: PropTypes.node
};

export default SideBar;
