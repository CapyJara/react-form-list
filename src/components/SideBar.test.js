import React from 'react';
import { shallow } from 'enzyme';
import SideBar from './SideBar';

describe('SideBar component', () => {
  it('renders a SideBar', () => {
    const wrapper = shallow(
      <SideBar>
        <a href={'o hey there'}>Testing</a>
      </SideBar> 
    );
    expect(wrapper).toMatchSnapshot();
  });
});
