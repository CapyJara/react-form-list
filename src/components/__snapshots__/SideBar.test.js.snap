// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBar component renders a SideBar 1`] = `
<nav>
  <a
    href="o hey there"
  >
    Testing
  </a>
</nav>
`;
